plugin/storage/es/samplingstore/dbmodel/model.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"time"
+
+	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/model"
+)
+
+type TimeThroughput struct {
+	Timestamp  time.Time           `json:"timestamp"`
+	Throughput []*model.Throughput `json:"throughputs"`
```

Comment:
It is not safe to depend on the `model.*` type for DB Objects - `model.*` is internal, it can change, which should not affect how JSON is stored inside of ES. I suggest re-defining fully internal data types here and converting from `model` to `dbmodel` fully.